app/controllers/application_controller.rb

```diff
@@ -11,5 +11,12 @@ class ApplicationController < ActionController::Base
   def current_user
     @current_user ||= User.where(id: session[:user_id]).first if session[:user_id]
   end
-  helper_method :current_user
+
+  def remove_cookie_banner(s)
```

Comment:
I would call this parameter something a bit more explicit like `markup`, or `text`.

Reply:
I caught it on linting. Renamed to content